UPGRADING.md

```diff
@@ -0,0 +1,179 @@
+    ]
+  }
+}
+```
+
+Then open `packages.json` and add this:
+```json
+"babel": {
+  "presets": [
+    "./webpack.babel.js"
+  ]
+}
+```
+
+Finally, download [webpackers babel preset](https://github.com/rails/webpacker/blob/master/package/babel/preset.js) file and place it in the same directory as `packages.json` with the name `webpack.babel.js`.
+
+## 2. Migrate from sass-rails to cssbundling-rails
+
+Start by following these steps:
+
+1. Add `cssbundling-rails` to your Gemfile (do not remove `sass-rails`, Sprockets still needs it);
+2. Run `./bin/bundle install`;
+3. Run `./bin/rails css:install:sass`.
+
+After you are done you will notice that once again the install step added various files to your project and updated some of the existing ones.
+
+**The new process 'Procfile.dev'**
+
+Just like the javascript process, this one will bundle and watch for changes in css files.
+
+**The 'build:css' attribute added to packages.json**
+
+This is the command `yarn build` will use to bundle css files.
+
+**The 'link_tree' directives removed from 'app/assets/manifest.js'**
+
+Now that the CSS files will be placed into `app/assets/build`, Sprockets no longer needs to worry about the `app/assets/stylesheets` folder. If you have any other `link_tree` for css files, remove them too.
+
+**Configuring multiple entrypoints**
+
+Sprockets will only compile files in the root directories listed in `manifest.js`, but the sass package that `yarn build` uses will also check subfolders, which might cause compilation errors if your scss files are using features like `@import` and variables. This means that if you have multiple entry points in your app, you have some extra work ahead of you. 
+
+Let's assume you have the following structure in your `app/asset/stylesheets` folder:
+
+```
+stylesheets/admin.scss
+stylesheets/admin/source_1.scss
+stylesheets/admin/source_2.scss
+stylesheets/application.csss
+stylesheets/application/source_1.scss
+stylesheets/application/source_2.scss
+```
+
+Start by your separating your entrypoints from your other files, and adjusting all `@import` for the new structure:
+
+```
+stylesheets/entrypoints/admin.scss
+stylesheets/entrypoints/application.scss
+stylesheets/sources/admin/source_1.scss
+stylesheets/sources/admin/source_2.scss
+stylesheets/sources/application/source_1.scss
+stylesheets/sources/application/source_2.scss
+```
+
+Then adjust the `build` attribute in `packages.json`:
+```
+"build:css": "sass ./app/assets/stylesheets/entrypoints:./app/assets/builds --no-source-map --load-path=node_modules"
+```
+
+**Deprecation warnings**
+
+Sass might raise deprecation warnings depending on what features you are using (such as division), but the messages will explain how to fix them. If you are not sure, see more details in the [official documentation](https://sass-lang.com/documentation/breaking-changes).
+
+## 3. Migrate from Sprockets to Propshaft
+
+Start by following these steps:
+
+1. Replace `sass-rails` with `propshaft`;
```

Comment:
Do we want to base this upgrade only on Rails 7 version? If not then sprockets migration needs some expanding for Rails 6, we would need to add section on how to remove sprockets from Rails < 7

Reply:
Is it even possible to remove Sprockets from Rails 6? Looking through [#43261](https://github.com/rails/rails/pull/43261/files) it seems it is part of the `gemspec` file and automatically imported

Reply:
Maybe remove was not the best word 😄 . It is possible not to require the sprockets, by explicitly requiring rails engines inside the `application.rb`.
But, my initial comment doesn't make sense since `propshaft` is rails 7+. Sorry :)